tests/devapps/KerberosConsole/AADKerberosLogger.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace KerberosConsole
+{
+    /// <summary>
+    /// Helper to provide internal tracking log for debugging.
+    /// Shows the message to the console and save to internal log file under 'Documents' folder.
+    /// </summary>
+    public static class AADKerberosLogger
+    {
+        private static int _bytesPerLine = 16;
+        private static int _lastFileIndex = 0;
+
+        /// <summary>
+        /// Name of log file.
+        /// </summary>
+        private static string _logFileName = "";
+
+        /// <summary>
+        /// Filename to save log data.
+        /// </summary>
+        internal static string LogFileName
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_logFileName))
+                {
+                    do
+                    {
+                        _lastFileIndex++;
+
+                        _logFileName = String.Format(
+                            CultureInfo.InvariantCulture,
+                            @"C:\Users\yoko\Documents\AzureAD-MSAL-{0}-{1}.log",

```

Comment:
Should probably point to the current windows user

Reply:
Nice catch. Test code replaced.